models/orders.py

```diff
@@ -2,11 +2,13 @@
 class Orders(db.Model):
    __tablename__ = 'Orders'
    OID = db.Column(db.Integer, primary_key=True, nullable=False)
-   MID = db.Column(db.Integer, foreign_key=True, nullable=False)
+   CMID = db.Column(db.Integer, foreign_key=True, nullable=False)
+   SMID = db.Column(db.Integer, foreign_key=True, nullable=False)
    Credit_num = db.Column(db.String(16), foreign_key=True)
```

Comment:
foreign_key要指標Credit card的number
改成這樣吧：
db.ForeignKey('<entity名稱>.<attr名稱>', onupdate="CASCADE", ondelete="CASCADE")

Reply:
I have modified all the files. By the way, in `app.py`, I switched to using dictionary formatting for the URL because it’s more convenient for us to modify. But it’s totally fine if you’d prefer to switch back.

Reply:
ok

Reply:
bro下次自己裝branch

Reply:
bro為啥刪掉其他branch